hack/tools/kind-control-config.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright 2020 IBM Corp.
+# SPDX-License-Identifier: Apache-2.0
+
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  extraPortMappings:
+  - containerPort: 30333  # Razee dash API node port in K8s
+    hostPort: 3333  # Razee dash API on host
+  - containerPort: 30080  # Razee dash UI node port in K8s
+    hostPort: 8080  # Razee dash UI on host
```

Comment:
Eventually we may not want to expose the UI to the outsiders but rather just port-forward to access it.

Reply:
Yes but I think for local development in kind it's easier to have this exposed right now.